gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/sql/execute/TablePrivilegeInfo.java

```diff
@@ -222,20 +227,48 @@ public void executeGrantRevoke( Activation activation,
 									List grantees)
 		throws StandardException
 	{
+		doExecuteGrantRevoke(activation, grant, grantees, columnBitSets, actionAllowed, true, td);
+		GemFireStore ms = Misc.getMemStore();
+		// TODO ashetkar Get a better way to check if security is enabled.
+		if (ms.isSnappyStore() && "LDAP".equalsIgnoreCase(ms.getBootProperty(Attribute.AUTH_PROVIDER))
```

Comment:
@ashetkar Isn't just the check whether property is present or not enough? This will not work when we add more auth types after 1.0. A better way to check is !(AuthenticationServiceBase.getPeerAuthenticationService() instanceof NoneAuthenticationServiceImpl). This can be added to AuthenticationServiceBase itself as a static method.